gprofiler/profilers/node.py

```diff
@@ -156,6 +155,7 @@ def _close_debugger(sock: WebSocket) -> None:
     }
     sock.send(json.dumps(cdp_request))
     sock.recv()
+    sock.close()
```

Comment:
There are 2 sites that use `create_debugger_socket` and both are not protecting from exceptions with regards to closing the socket. Let's change the sites `with create_debugger_socket(...)` and ensure that the object's context manager actually closes it after the `with`. This ensures no leaks even if exceptions occur.

Reply:
I will add it in separate PR.

Reply:
Is it ok :D ?

Reply:
Yes

Reply:
Did you open an issue for it, or you remember to fix?

Reply:
https://github.com/Granulate/gprofiler/issues/576 -> just opened it

Reply:
Excellent